bindings/java/src/org/sleuthkit/datamodel/IngestJobInfo.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Sleuth Kit Data Model
+ *
+ * Copyright 2011-2016 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *	 http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.datamodel;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Represents information for an ingest job.
+ */
+public final class IngestJobInfo {
+
+	public enum IngestJobStatusType {
+
+		//DO NOT CHANGE ORDER
+		STARTED,
+		CANCELLED,
+		COMPLETED;
+
+		public static IngestJobStatusType fromID(int typeId) {
+			for (IngestJobStatusType statusType : IngestJobStatusType.values()) {
+				if (statusType.ordinal() == typeId) {
+					return statusType;
+				}
+			}
+			return null;
+		}
+	}
+
+	private final int ingestJobId;
```

Comment:
Let's go ahead and make the ingestJobId field a long as we do with most ids, just for consistency. It might make life slightly easier for developers.